.github/release/release_notes.md

```diff
@@ -1,3 +1,74 @@
+# [v0.1.9](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.9)
+
+## Force interactive mode for `update` command
+
+Interactive mode for `update` sub-command is now enabled by default. 
+
+Added in https://github.com/VirtusLab/scala-cli/pull/1100 by @lwronski
+
+## `--list-main-classes` for `publish` & `package`
+
+`publish` and `package` sub-commands handle `--list-main-classes` option, which lists all the available main classes.
+
+Added in https://github.com/VirtusLab/scala-cli/pull/1118  by @Gedochao
+
+## Fixes
+
+### `fmt` options improvement
+
+It is now possible to pass native `scalafmt` options in `fmt` sub-command with the `-F`. 
+```
+$ scala-cli fmt -F --version
+scalafmt 3.5.2
+```
+
+`--respect-project-filters` is turned on by default in Scala CLI. This addresses https://github.com/VirtusLab/scala-cli/issues/1121
```

Comment:
```suggestion
`--respect-project-filters` is an alias for `-F --respect-project-filters`. Because of the way sources are passed by Scala CLI to `scalafmt` under the hood, we now turn it on by default to respect any `project.excludePaths` settings in the user's `.scalafmt.conf`.  
It can be disabled by passing `--respect-project-filters=false` to revert to previous behaviour.

This addresses https://github.com/VirtusLab/scala-cli/issues/1121
```